MarkdownToLatex/MarkdownToLatex/MdToTex.cs

```diff
@@ -57,7 +57,7 @@ internal static void convertMathElement(Match match)
         internal static void convertText(string text)
         {
             string tempstr = text.TrimStart();
-            string bcText = LatexRenderer.WriteCursive(LatexRenderer.WriteBold(text));
+            string bcText = LatexRenderer.WriteVerbatim(LatexRenderer.WriteInlineSyntax(text));
```

Comment:
Ah du machst das so, ich dachte das alles passiert in LatexRenderer und hier wird einfach nur `LatexRenderer.WriteInlineSyntax(text)` aufgerufen. Willst du das so lassen?

Reply:
Oh du hast recht, kleinen Moment